### DISCUSSION TITLE
'content-editable' inspector input type doesn't work properly

### DISCUSSION THREAD

**Discussion by pandika-csgi** [clientIO/joint#2256] [Q&A]
### Introduction

Hi, previously using JointJS v3.6.5 trial version (work properly), currently using v3.7.1 (27/06/2023) licensed version

My app is broken, especially with `content-editable` input, I try to update the input but input doesn't update to new data

I try to replicate using new input with the same `content-editable` type but yeah input doesn't change, is that bug?
I'm still investigating if is there any issue with my code while waiting for confirmation from JointJS 

Thanks

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by pandika-csgi** [comment]
I found this https://github.com/clientIO/joint/issues/1487, I try the same thing

First attempt:
- Remove `package.lock.json`
- Remove `node_modules` folder
- `npm cache clean --force`
- npm install

Second attempt:
- Update jquery from 3.6.0 to 3.7.0
- Remove `package.lock.json`
- Remove `node_modules` folder
- `npm cache clean --force`
- npm install

Last attempt:
- Install jquery from 3.6.0 to 3.6.4 (I've seen in rappid project using this version)
- Remove `package.lock.json`
- Remove `node_modules` folder
- `npm cache clean --force`
- npm install

Unfortunately, the issue still occurred

**Comment by pandika-csgi** [comment]
I try to remove `jquery` package from `package.json` and run the same thing 

- Remove package.lock.json
- Remove node_modules folder
- npm cache clean --force
- npm install

and it's working!
Thus rappid already included jquery and don't need to add it separately? @kumilingus

**Reply by kumilingus** [reply]
Correct, the `rappid.tgz` package installs all the dependencies for you.

**Reply by pandika-csgi** [reply]
Thanks for the confirmation!
